src/EventStore.Core/LogV3/LogV3SystemEventTypes.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using EventStore.Core.Services;
+
+namespace EventStore.Core.LogV3
+{
+	public class LogV3SystemEventTypes {
+
+		public const UInt32 FirstRealEventType = 1024;
+		public const UInt32 EventTypeInterval = 1;
+
+		public const UInt32 EmptyEventType = 0;
```

Comment:
im not sure if we need all of these, generally we only need ones that are hard coded
i.e. we refer to by number in the codebase

i think worth adding 'number' to the end same as the streams equivalent to make these easier to find in the code base